lib/carin_for_blue_button_test_kit/client/v2.0.0/c4bb_client_test_suite.rb

```diff
@@ -106,41 +104,89 @@ def self.test_resumes?(test)
     group do
       run_as_group
       title 'Wait for Claims Data and Search Requests'
+      description %(
+        This test will wait for the client system under test to submit requests for resources for each Carin for Blue
+        Button profiles, and for requests including all of the required search parameters for each resource type.
+      )
+
       test from: :initial_wait_test
     end
 
     group do
       title 'C4BB Patient Profile claims data and search request tests'
+      description %(
+        These tests look through all the requests sent in from the client system to ensure a request was made that
+        returned the Patient resource implementing the Carin for Blue Button Patient profile from the Inferno
+        Reference Server. It will also verify that each required search Parameter for the Patient resource specified
+        in the Carin for Blue Button IG and the US Core 3.1.1 IG  was used in requests made by the client.
+      )
+
```

Comment:
Here and in many other places in the test kit
- correct Carin -> CARIN
- add explicit links to profiles and other resources (e.g., capability statements defining search parameter requirements)